### PR TITLE
Update examples/extension/README.md

### PR DESCRIPTION
**Pull request by crossoverJie**
Update `build.grade` URL.

### CODE REVIEW

**Comment by laurit** [comment]
While the idea of linking to the exact line is nice, the problem with it is that the link will point to a fixed version. If the file is changed, the link ends up pointing to an old version that could be different from the lates, which could be confusing.

**Comment by crossoverJie** [comment]
You are right, but even if you link to the main branch, the number of lines of code may change😂.

**Comment by laurit** [comment]
> You are right, but even if you link to the main branch, the number of lines of code may change😂.

Perhaps instead of linking to the line we could use 
https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/examples/extension/build.gradle#:~:text=extendedAgent Though this is not supported in all browsers https://caniuse.com/url-scroll-to-text-fragment

**Comment by crossoverJie** [comment]
Good suggestion.

File: examples/extension/README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ contain extension jars) for the `otel.javaagent.extensions` value.
 
 To simplify deployment, you can embed extensions into the OpenTelemetry Java Agent to produce a single jar file. With an integrated extension, you no longer need the `-Dotel.javaagent.extensions` command line option.
 
-For more information, see the `extendedAgent` task in [build.gradle](build.gradle).
+For more information, see the `extendedAgent` task in [build.gradle](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/examples/extension/build.gradle?rgh-link-date=2024-03-11T10%3A36%3A52Z#:~:text=extendedAgent).
```
[the remark this answers is not in the document]
Comment (crossoverJie) [reply examples/extension/README.md]: Done.

[the remark this answers is not in the document]
Comment (laurit) [reply examples/extension/README.md]: why do you need to add `?T10%3A36%3A52Z` to the link?

[the remark this answers is not in the document]
Comment (crossoverJie) [reply examples/extension/README.md]: I was careless.
Done.

**Review by laurit — APPROVED** [review]

**Pull request merged**